src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/SearchNavSideMenuComponentObject.java

```diff
@@ -0,0 +1,153 @@
+	@FindBy(css = ".local-wikia-search > li > a")
+	private WebElement searchResults;
+	@FindBy(css = "li[class='back']")
+	private WebElement menuBackButton;
+	@FindBy(css = ".icon chevron")
+	private WebElement needsClickChevrons;
+	@FindBy(css = ".local-wikia-search a")
+	private List<WebElement> searchSuggestions;
+	@FindBy(css = ".local-wikia-search span")
+	private WebElement searchAlerts;
+	@FindBy(css = ".search")
+	private WebElement searchView;
+	@FindBy(css = "div[class='drawer']")
+	private WebElement menuView;
+	@FindBy(css = ".local-nav-menu > li > div")
+	private List<WebElement> chevrons;
+	@FindBy(css = ".local-nav-menu > li > a")
+	private List<WebElement> noChevrons;
+	@FindBy(css= ".back")
+	private WebElement backChevron;
+	@FindBy(css = ".overlay")
+	private WebElement overlay;
+
+	public SearchNavSideMenuComponentObject(WebDriver driver) {
+		super(driver);
+	}
+
+	public void clickSearchField() {
+		waitForElementByElement(searchInput);
+		searchInput.click();
+		PageObjectLogging.log("clickSearchField", "Search field was clicked", true);
+	}
+
+	public MercuryBasePageObject clickSearchSuggestion(int index) {
+		waitForElementByElement(searchSuggestions.get(index));
+		searchSuggestions.get(index).click();
+		return new MercuryBasePageObject(driver);
+	}
+
+	public void clickCancelButton() {
+		waitForElementVisibleByElement(cancelSearchCaption);
+		cancelSearchCaption.click();
+	}
+
+	public String getSearchResultHref(int searchPosition) {
+		waitForElementVisibleByElement(searchSuggestions.get(searchPosition));
+		return searchSuggestions.get(searchPosition).getAttribute("href");
+	}
+
+	public void typeInSearchField(String content, int length) {
+		waitForElementVisibleByElement(searchInput);
+		searchInput.sendKeys(content.substring(0, length));
+		PageObjectLogging.log(
+				"typeInSearchField",
+				"String >>" + content.substring(0, length) + "<< was typed in string field",
+				true
+		);
+	}
+
+	public void verifySearchSuggestionsWereVisible() {
+		waitForElementByElement(searchSuggestions.get(0));
+		Assertion.assertTrue(checkIfElementOnPage(searchSuggestions.get(0)));
+		PageObjectLogging.log("verifySearchSuggestionsWereVisible", "Search suggestions were visible", true, driver);
+	}
+
+	public void verifySearchView() {
+		Assertion.assertTrue(checkIfElementOnPage(searchView));
+		PageObjectLogging.log("verifySearchView", "Search view was displayed on full size", true, driver);
+	}
+
+	public void verifyMenuView() {
+		Assertion.assertTrue(checkIfElementOnPage(menuView));
+		PageObjectLogging.log("verifyMenuView", "Menu view was visible", true, driver);
+	}
+
+	public void verifySearchNotMatch() {
+		waitForElementByElement(searchAlerts);
+		Assertion.assertTrue(searchAlerts.getText().contains(MercuryContent.MERCURY_SEARCH_NOT_MATCH));
+	}
+
+	public void verifyOpeningArticleInNav(int anchorIndex) {
+		waitForElementVisibleByElement(noChevrons.get(anchorIndex));
+		if (noChevrons.size() != 0) {
```

Comment:
is this check necessary since noChevrons.get(index) is already being called prior to this?